src/Components/jinsubaek/CommentLists/CommentLists.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './CommentLists.scss';
+
+class CommentsList extends React.Component {
+  handleClickDeleteButton(commentId) {
+    this.props.deleteComment(commentId);
+  }
+
+  render() {
+    const { comment } = this.props;
+    return (
+      <li>
+        <div>
+          <span className="name">{comment.userName}</span>
+          <span>{comment.content}</span>
+          <button
+            className="delete"
+            onClick={() => this.handleClickDeleteButton(comment.id)}
```

Comment:
- 추가적인 로직이 필요한 경우가 아닌 이상 새롭게 함수를 생성할 이유가 없어보입니다.
```suggestion
            onClick={() => this.props.deleteComment(comment.id)}
```

Reply:
감사합니다. 수정했습니다.